public/server/db/dae.json

```
@@ -0,0 +1,3 @@
+{
+  "definition": "Does anyone else"
```

The first letter of each word should be in caps.
```suggestion
  "definition": "Does Anyone Else"
```